plugins/jobs/plugin.zsh

```diff
@@ -0,0 +1,14 @@
+geometry_prompt_jobs_setup() {
+  return true
```

Comment:
I like true/false better than 0/1, we should start using this convention in more places